pkg/config/application_kubernetes.go

```diff
@@ -34,6 +34,19 @@ type KubernetesApplicationSpec struct {
 	Workloads []K8sResourceReference `json:"workloads"`
 	// Which method should be used for traffic routing.
 	TrafficRouting *KubernetesTrafficRouting `json:"trafficRouting"`
+
+	// The label key of an used by selector.
+	// Default is pipecd.dev/variant.
+	VariantLabelKey string `json:"variantLabelKey"`
```

Comment:
Use default tag to set its default value.

```suggestion
	VariantLabelKey string `json:"variantLabelKey" default:"pipecd.dev/variant"`
```